io.parmentier.flatpak.streamdeck.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.parmentier.flatpak.streamdeck
+base-version: '5.15'
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: streamdeck
+finish-ars:
+  - --socket=x11
+  - --socket=session-bus
```

Comment:
if not present, it does not appear on the task manager/bar.

Reply:
pass `--log-session-bus` to `flatpak run` to determine what it needs.